index.html

```diff
@@ -9,17 +9,37 @@
     <title>CodersCamp2020 | CookBook</title>
     <link rel="stylesheet" href="styles/App.css" />
 
-    <link
-        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-        rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/HomePage.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
 </head>
 
 <body>
 
-    <div id="swquiz-app">
-    </div>
+    <div id="swquiz-app" class="homePageContaiiner">
+        <div class="searchDescription">
+            Look for a meal by the ingredients you already have
+        </div>
+        <div class="searchSection">
+            <div id="tags"></div>
+            <div class="inputWrapper">
+                <form>
+                    <input id="ingredient" type="text" placeholder="e.g. pasta, tuna" name="search">
+                </form>
+                <button id="searchButton"><i class="fa fa-search"></i></button>
```

Comment:
Move button into the form. It will work not only when button clicked but also on enter